### ISSUE TITLE
lerna command execute success but exit with node error

### ISSUE THREAD

**Issue by kagawagao** [lerna/lerna#2373]
## Expected Behavior

exit with no error
## Current Behavior

```
lerna success published 5 packages
node[71853]: ../src/signal_wrap.cc:159:void node::DecreaseSignalHandlerCount(int): Assertion `(new_handler_count) >= (0)' failed.
 1: 0x100b8ff9a node::Abort() (.cold.1) [/usr/local/bin/node]
 2: 0x1000832c4 node::FatalError(char const*, char const*) [/usr/local/bin/node]
 3: 0x10008307c node::AppendExceptionLine(node::Environment*, v8::Local<v8::Value>, v8::Local<v8::Message>, node::ErrorHandlingMode) [/usr/local/bin/node]
 4: 0x1000f7e52 node::HasSignalJSHandler(int) [/usr/local/bin/node]
 5: 0x1000f85ad node::(anonymous namespace)::SignalWrap::Close(v8::Local<v8::Value>) [/usr/local/bin/node]
 6: 0x10003eb76 node::Environment::CleanupHandles() [/usr/local/bin/node]
 7: 0x10003ed2f node::Environment::RunCleanup() [/usr/local/bin/node]
 8: 0x1000b4a3a node::NodeMainInstance::Run() [/usr/local/bin/node]
 9: 0x100060f13 node::Start(int, char**) [/usr/local/bin/node]
10: 0x7fff6377e7fd start [/usr/lib/system/libdyld.dylib]
[1]    71531 abort      npm run pub
```
## Possible Solution

compatible with `Node v13.2.0`
## Steps to Reproduce (for bugs)

1. install `Node v13.2.0`
2. execute `lerna` command

<details><summary>lerna.json</summary><p>

```json
{
  "packages": ["packages/*"],
  "version": "independent",
  "ignoreChanges": ["**/*.md", "**/__tests/**", "**/package-lock.json"]
}
```
</p></details>

<details><summary>lerna-debug.log</summary><p>

</p></details>

## Your Environment

| Executable | Version |
| ---: | :--- |
| `lerna --version` | `3.19.0` |
| `npm --version`  | `6.13.2` |
| `yarn --version` |`1.19.2` |
| `node --version` | `v13.2.0` |

| OS | Version |
| --- | --- |
| macOS Catalina | `10.15.2` |

**Comment by Ahkari** [comment]
same problem

**Comment by zemd** [comment]
same problem for 13.2.0, but seems fixed by upgrading node up to 13.3.0
